api_v3/lib/types/entry/KalturaLiveEntry.php

```diff
@@ -36,10 +36,17 @@
 	/**
 	 * Elapsed recording time (in msec) up to the point where the live stream was last stopped (unpublished).
 	 * @var int
+	 * @deprecated
 	 */
 	public $lastElapsedRecordingTime;
```

Comment:
@AviBueno, I think this field is used internally by Kaltura servers and you can remove it and break compatibility, it won't affect anyone.